### PR TITLE
Added tracking issue, updated deprecation notice for interleaved tables

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes https://github.com/cockroachdb/docs/issues/9267.

### CODE REVIEW

File: v20.2/interleave-in-parent.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ Interleaving tables does not affect their behavior within SQL.
 {{site.data.alerts.end}}
 
 {{site.data.alerts.callout_danger}}
-Interleaved tables will be deprecated in a future release. After [upgrading to v20.2](upgrade-cockroach-version.html), we recommend that you [convert any existing interleaved tables to non-interleaved tables](#convert-interleaved-tables-to-non-interleaved-tables) and replace any existing interleaved secondary indexes with non-interleaved indexes. We do not recommend interleaving tables in new clusters.
+Interleaved tables will be deprecated in a future release, as their performance benefits are limited to a small number of use cases that no longer justify the cost in engineering resources for maintenance.
```
Comment (vy-ton) [inline v20.2/interleave-in-parent.md:15]: We probably want to tweak the language here to provide more context and indicate the performance benchmarking that was completed. There's some ongoing discussion with the SQL Schema team on how to convey this

**Comment by jseldess** [comment]
@ericharmeling, do you have plans to pick this back up, or is it out-dated?

**Comment by ericharmeling** [comment]
> @ericharmeling, do you have plans to pick this back up, or is it out-dated?

Vy and I put this on hold, pending a decision on whether we want to make our performance benchmarking public. IIUC, we plan to pick this up after that decision is made. Is that correct, @vy-ton ?

**Comment by ericharmeling** [comment]
Closing, in favor of https://github.com/cockroachdb/docs/pull/10154.

**Pull request closed without merging**